charts/dex/templates/ingress.yaml

```diff
@@ -1,6 +1,9 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "dex.fullname" . -}}
 {{- $svcPort := .Values.service.ports.http.port -}}
+{{- if .Values.https.enabled -}}
```

Comment:
I'd like to avoid forcing HTTPS there. Let's make it optional.

Reply:
@nabokihms Really would appreciate this being merged in, maybe some more information on whether we should add a flag to the ingress to use HTTPS port?